app/blueprints/assessments/helpers.py

```diff
@@ -265,4 +265,10 @@ def sanitise_export_data(data):
 
 def exclude_header(fund_names: list):
     if "COF-EOI" in fund_names:
-        return ["Help with insolvency", "Help with organisation type", "Help with public authority"]
+        return [
+            "Help with insolvency",
+            "Help with organisation type",
+            "Help with public authority",
+            "Help gydag awdurdod cyhoeddus",
+            "Help gyda'r math o sefydliad",
```

Comment:
I thought this PR was removing these redundant columns. I still see these columns being added still?

Unless I am misinterpreting your code change?

@ramsharma-prog

Reply:
This function is called to excludes these columns 

https://github.com/communitiesuk/funding-service-design-assessment/blob/5e3e07cf6ddf0fc14d4fd9c837ee51fd74d14c39/app/blueprints/assessments/helpers.py#L167